src/toil/job.py

```diff
@@ -993,7 +993,7 @@ def _createEmptyJobGraphForJob(self, jobStore, command=None, predecessorNumber=0
         """
         # set _config to determine user determined default values for resource requirements
         self._config = jobStore.config
-        return jobStore.create(JobNode.fromJob(self, command=command,
+        return jobStore.getJobGraph(JobNode.fromJob(self, command=command,
                                                predecessorNumber=predecessorNumber))
```

Comment:
I hate making style comments, but this one jumped out at me: the indent isn't aligned anymore.